### PR TITLE
Api restaurant show page 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
## Description
- Added serializer for menu, product categories and products
- Testing the endpoints for a specific restaurant
- Added rack-cors gem for middleware management

### CODE REVIEW

File: app/serializers/product_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ProductSerializer < ActiveModel::Serializer
+  attributes :name, :price
```
Comment (oliverochman) [inline app/serializers/product_serializer.rb]: add image file link attribute?

File: app/serializers/restaurants/index_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Restaurants::IndexSerializer < ActiveModel::Serializer
+  attributes :id, :name
```
Comment (oliverochman) [inline app/serializers/restaurants/index_serializer.rb:2]: longitude and latitude?

File: spec/requests/api/restaurants_endpoint_spec.rb [hunk 3]
```diff
@@ -39,31 +36,65 @@
...
+      let!(:product_category) {create(:product_category,
+                                      name: 'Main courses',
+                                      menu: menu_lunch,
+                                      restaurant: thai_food)}
+      let!(:lunch_box){ create(:product, name: 'Lunch Box',
+                               description: 'Marvelous lunch',
+                               price: 75,
+                               product_category: product_category,
+                               restaurant: thai_food,
+                               image_file_link: 'http://example.com/image.jpg')}
```
Comment (oliverochman) [inline spec/requests/api/restaurants_endpoint_spec.rb:69]: why do you have this attribute here but not in the serializer?

File: app/serializers/restaurants/index_serializer.rb (hunk 2 above)
Comment (SophieSjoberg) [reply app/serializers/restaurants/index_serializer.rb:2]: We are not testing for that yet

**Review by NegativeKarma — COMMENTED** [review]
LGTM, we need a final opinion @tochman @diraulo

**Review by diraulo — APPROVED** [review]
Looks good to me

**Pull request merged**